### PR TITLE
[FIX] avoid stall on Windows when TOPPAS calls a TOPP tool with a ver…

### PR DESCRIPTION
**Pull request by cbielow**
…y long list of input files (exceeding the 8k length limit)

Fixed by moving the list of files into the .ini file.

### CODE REVIEW

File: src/openms_gui/source/VISUAL/TOPPASToolVertex.cpp [hunk 1]
```diff
@@ -525,19 +525,22 @@ namespace OpenMS
 
         String param_name = in_params[param_index].param_name;
 
+        const QStringList& file_list = ite->second.filenames.get();
+
         bool store_to_ini = false;
         // check for GenericWrapper input/output files and put them in INI file:
-        if (param_name.hasPrefix("ETool:"))
+        // OR if there are a lot of input files (which might exceed the 8k length limit of cmd.exe on Windows)
+        if (param_name.hasPrefix("ETool:") || file_list.size() > 10)
```
Comment (timosachsenberg) [inline src/openms_gui/source/VISUAL/TOPPASToolVertex.cpp]: can you make the threshold a constant?

Comment (timosachsenberg) [reply src/openms_gui/source/VISUAL/TOPPASToolVertex.cpp]: as it is reused in other parts of the code

**Comment by cbielow** [comment]
done

**Comment by timosachsenberg** [comment]
src/openms_gui/source/VISUAL/TOPPASToolVertex.cpp:539:64: error: comparison between signed and unsigned integer expressions [-Werror=sign-compare]

**Pull request merged**